makefile

```diff
@@ -367,18 +367,29 @@ usbflash :
 	tools/reflash_and_monitor.bat
 else
 usbflash :
```

Comment:
Tested on Windows too?

Reply:
Will test later - this is not ready for merge.